test/test_all.dart

```diff
@@ -41,6 +43,7 @@ void main() {
   multireddit_test.main();
   redditor_test.main();
   rising_listing_mixin_test.main();
+  best_listing_mixin_test.main();
```

Comment:
General comment: sort these by alphabetical order (`live_auth_test` is sort of an exception).